main/src/ca/uwaterloo/flix/language/phase/Verifier.scala

```diff
@@ -28,7 +28,7 @@ object Verifier {
 
   def run(root: Root)(implicit flix: Flix): Root = flix.phase("Verifier") {
     if (!flix.options.xnoverify) {
```

Comment:
Could we change this to early return?

Reply:
I wrote it like this to be consistent with Optimizer - I can change both? They can also both just use if-else

Reply:
Yes, please